Gem/Code/Source/Components/Multiplayer/GemSpawnerComponent.cpp

```diff
@@ -20,9 +20,10 @@ namespace MultiplayerSample
         if (serializeContext)
         {
             serializeContext->Class<GemSpawnable>()
-                ->Version(1)
+                ->Version(2)
                 ->Field("Tag", &GemSpawnable::m_tag)
                 ->Field("Asset", &GemSpawnable::m_gemAsset)
+                ->Field("Score", &GemSpawnable::m_scoreValue)
```

Comment:
"Value" would also work, but maybe that seemed to vague?

Reply:
Yeah, that was my thought.